src/parser/jobs/ast/modules/ArcanaTracking/ArcanaDoTsGroup.ts

```diff
@@ -0,0 +1,103 @@
+import {ActionRoot} from 'data/ACTIONS/root'
+import {StatusRoot} from 'data/STATUSES'
+//this module is used to store all delayed attacks/dots relevant for card plays
+//only actions are tracked here as the related statuses can be looked up using statusesApplied
+
+export const DOTS_ALIAS: [[keyof StatusRoot, keyof StatusRoot]] = [
+	//these dots are tracked via other statuses rather than the original afflicted status. each with their own reasoning below
+
+	//MCH wildfire is reapplied with each weaponskill applied. much easier to just track the wildfire status on self for this purpose
+	['WILDFIRE', 'WILDFIRE_SELF'],
+]
+
+export const DOTS_AND_GROUND_ACTIONS: Array<keyof ActionRoot> = [
+
+	//**MELEE**
+	//DRG
+	'CHAOS_THRUST',
+	'CHAOTIC_SPRING',
+
+	//NIN
+	'DOTON',
+	'DOTON_TCJ',
+
+	//RPR
+	//none for RPR. Comment left here as reminder to review in future patches
+
+	//MNK
+	'DEMOLISH',
+
+	//SAM
+	'HIGANBANA',
+	'KAESHI_HIGANBANA',
+
+	//**PHYS RANGED**
+	//BRD
+	'VENOMOUS_BITE',
+	'WINDBITE',
+	'IRON_JAWS',
+	'CAUSTIC_BITE',
+	'STORMBITE',
+
+	//MCH
+	'FLAMETHROWER',
```

Comment:
Important to note here that ground DoTs like FT and Doton don't snapshot debuffs like Chain Strat - I think they need a slightly different code path.

Reply:
Updated! Thanks for this
I've moved all ground dots to a separate const with separate handling
See below commit